<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardManual</class>
 <widget class="QWidget" name="WizardManual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ManualLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16pt;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Manual Import</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FilePathEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>360</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="GamePathInstruction">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Enter the games path and name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>360</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="GameNameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>225</y>
     <width>360</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="GameNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Game Name:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
